onnxscript/function_libs/torch_lib/ops/core.py

```diff
@@ -8372,6 +8372,52 @@ def aten_unique_consecutive(
     raise NotImplementedError()
 
 
+@torch_op("aten::unique", trace_only=True)
+def aten_unique(
+    self: TensorType,
+    sorted: bool = True,
+    return_inverse: bool = False,
+    return_counts: bool = False,
+    dim: Optional[int] = None,
+) -> tuple[TensorType, TensorType, TensorType]:
+    """unique(Tensor self, bool sorted=True, bool return_inverse=False, bool return_counts=False, int? dim=None) -> (Tensor, Tensor?, Tensor?)"""
+
+    if dim is None:
```

Comment:
Codecov reported that this function is not covered. Is there a way to unit test this function?

Reply:
I think this is a false positive. The function is definitely tested as shown by the assertion error indicated in https://github.com/microsoft/onnxscript/pull/1547#discussion_r1607006325 that arises if one changes it.

Reply:
This function no longer exists.